### PR TITLE
Update collection metadata - Mobile

### PR DESCRIPTION
**Pull request by jakzaizzat**
### Summary of Changes

Added a feature to allow user submit collection request changes form. 

### Demo or Before/After Pics

https://github.com/gallery-so/gallery/assets/4480258/2f3f29d0-7fa8-45ec-a9b8-c220848bbb3d

**Error message**
CleanShot 2024-02-01 at 11 50 53@2x

### Edge Cases

1. Best case: Able to submit the form
2. Worst case: Unable to submit the form (should show error message)

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/components/Community/CommunityMetadataFormBottomSheet.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        `https://formspree.io/f/${env.EXPO_PUBLIC_FORMSPEE_REQUEST_COLLECTION_ID}`,
+        {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            communityId: community.dbid,
+            communityName: community.name,
+            message,
```
Comment (kaitoo1) [inline apps/mobile/src/components/Community/CommunityMetadataFormBottomSheet.tsx:66]: same comment as web - could you include the submitter's user dbid 🙏

File: apps/mobile/src/components/Community/CommunityMetadataFormBottomSheet.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+                font={{ family: 'ABCDiatype', weight: 'Regular' }}
+              >
+                Something went wrong! Please try again
+              </Typography>
+            )}
+          </View>
+
+          <Button
+            onPress={handleSubmitPress}
+            text="SUBMIT"
```
Comment (kaitoo1) [inline apps/mobile/src/components/Community/CommunityMetadataFormBottomSheet.tsx:145]: lets disable the button if there isn't any input

**Review by kaitoo1 — APPROVED** [review]
LGTM! just a few small requests 🙏

File: apps/mobile/src/components/Community/CommunityMetadataFormBottomSheet.tsx (hunk 1 above)
Comment (jakzaizzat) [reply apps/mobile/src/components/Community/CommunityMetadataFormBottomSheet.tsx:85]: Added!

File: apps/mobile/src/components/Community/CommunityMetadataFormBottomSheet.tsx (hunk 2 above)
Comment (jakzaizzat) [reply apps/mobile/src/components/Community/CommunityMetadataFormBottomSheet.tsx:168]: Updated!

**Pull request merged**